yarn.lock

```diff
@@ -938,11 +938,6 @@
   resolved "https://registry.yarnpkg.com/@xtuc/long/-/long-4.2.2.tgz#d291c6a4e97989b5c61d9acf396ae4fe133a718d"
```

Comment:
Do we need yarn.lock - is it because of this morning's breaking drive?

Reply:
as discussed, we keep it in for now